pkg/shell/cockpit-plot.js

```diff
@@ -24,7 +24,128 @@ define([
     "shell/cockpit-main"
 ], function($, cockpit, shell) {
 
-function setup_plot(graph_id, resmon, data, user_options, store_samples) {
+function setup_plot(graph_id, grid, data, user_options) {
+    var options = {
+        colors: [ "#0099d3" ],
+        legend: { show: false },
+        series: { shadowSize: 0,
+                  lines: { lineWidth: 0.0,
+                           fill: 1.0
+                         }
+                },
+        xaxis: { tickFormatter: function() { return ""; } },
+        yaxis: { tickFormatter: function() { return ""; } },
+        // The point radius influences
+        // the margin around the grid
+        // even if no points are plotted.
+        // We don't want any margin, so
+        // we set the radius to zero.
+        points: { radius: 0 },
+        grid: { borderWidth: 1,
+                aboveData: true,
+                color: "black",
+                borderColor: $.color.parse("black").scale('a', 0.22).toString(),
+                labelMargin: 0
+              }
+    };
+
+    var num_series = grid.rows.length;
```

Comment:
This is unused and a bit confusing.  grid.rows.length is 8, for example, while data.length is only 4.